packages/payload/src/auth/operations/me.ts

```diff
@@ -41,6 +41,7 @@ export const meOperation = async (args: Arguments): Promise<MeOperationResult> =
 
     if (user) {
       user.collection = collection.config.slug
+      user._strategy = req.user._strategy
```

Comment:
Should we mark `result.strategy` as deprecated? No sense in returning this property twice. The deprecated flag could point to here as the new source of truth going forward.